src/ssl.c

```diff
@@ -4053,6 +4053,29 @@ int wolfSSL_GetHmacSize(WOLFSSL* ssl)
     return BAD_FUNC_ARG;
 }
 
+int wolfSSL_GetPeerSequenceNumber(WOLFSSL* ssl, word64 *seq)
```

Comment:
This code could be a problem for some smaller embedded targets. Can you wrap this in `#ifndef NO_64BIT` or `#ifdef WORD64_AVAILABLE`. Either works.

Reply:
Added `#ifdef WORD64_AVAILABLE`